infrastructure/iac/iac.go

```diff
@@ -385,8 +409,56 @@ func (iac *Scanner) toAdditionalData(affectedFilePath string, issue iacIssue) sn
 		Issue:         issue.IacDescription.Issue,
 		Impact:        issue.IacDescription.Impact,
 		Resolve:       issue.IacDescription.Resolve,
-		Path:          issue.Path,
+		Path:          iacIssuePath,
 		References:    issue.References,
+	}, nil
+}
+
+func (iac *Scanner) parseIacIssuePath(path []interface{}) ([]string, error) {
+	var pathTokens []string
+	for _, p := range path {
+		pt, err := newIacIssuePathToken(p, iac.errorReporter)
```

Comment:
You could avoid passing error reporter down to the `iacIssuePathToken` struct by capturing error using error reporter within this function.

Reply:
Or just make it a caller function, e.g. func (iac ...) ...

Reply:
Changed in https://github.com/snyk/snyk-ls/pull/269/commits/d53881ee92568432ace2f313971cb3a49ed4c491